client/components/domains/register-domain-step/index.jsx

```diff
@@ -273,7 +273,7 @@ class RegisterDomainStep extends React.Component {
 			<div className="register-domain-step">
 				<div className="register-domain-step__search">
 					<SearchCard
-						ref="searchCard"
+						ref={ searchCard => ( this.searchCard = searchCard ) } // eslint-disable-line react/jsx-no-bind
```

Comment:
Why not declare a helper method to do this?
```
storeSearchCardRef = searchCard => {
    this.searchCard = searchCard;
}
<SearchCard ref={ this.storeSearchCardRef } /* ... */ />
```
Then there won't be a need for eslint exceptions :)